tripy/tests/integration/test_batchnorm.py

```diff
@@ -0,0 +1,59 @@
+import torch
+import pytest
+import re
+
+import tripy as tp
+from tests import helper
+
+DTYPES = [(torch.float32, tp.float32), (torch.float16, tp.float16)]
+
+class TestBatchNorm:
+
+    @pytest.mark.l1
```

Comment:
How slow are these tests?

Reply:
they are like 10-12 seconds, should I remove `l1`?

Reply:
Can you try doing the reference computation and comparison on GPU? The device -> host copy adds unnecessary overhead.

Reply:
done, and removed L1